src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -159,6 +218,29 @@ private InlineDiffLine<CodeLine>[] CreateDiffOnlyLines(InlineDiffLine<CodeLine>[
                 }
             }
             return filteredLines.ToArray();
+
+            // The lines of code are already marked up with HTML which makes
+            // them a little difficult to process.  This strips out all of the
+            // markup and computes the size of the whitespace indent at the
+            // beginning of the line.
+            static (int indentSize, string code) GetIndentAndCode(string line)
+            {
+                if (string.IsNullOrEmpty(line)) { return (0, null); }
```

Comment:
```suggestion
                if (string.IsNullOrEmpty(line)) { return (0, ""); }
```

Reply:
Good suggestion.  I'll also add a check to not include empty strings as part of the context so these are effectively ignored when there are empty lines.